### PR TITLE
Rename set() to store() for kt singletons

### PR DESCRIPTION
**Pull request by SHeimlich**

### CODE REVIEW

File: particles/Tutorial/Kotlin/Demo/src/total/TTTHumanPlayer.kt [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.tutorials.tictactoe
+
+import arcs.sdk.Handle
+
+class TTTHumanPlayer : AbstractTTTHumanPlayer() {
```
Comment (piotrswigon) [inline particles/Tutorial/Kotlin/Demo/src/total/TTTHumanPlayer.kt]: Diff shows this file being introduced with this patch, which doesn't seem right. Any ideas why?

Comment (SHeimlich) [reply particles/Tutorial/Kotlin/Demo/src/total/TTTHumanPlayer.kt]: No idea how it snuck in... I got rid of it.

**Comment by SHeimlich** [comment]
Closed in favor of #4723

**Pull request closed without merging**